tox.ini

```diff
@@ -20,6 +20,7 @@ deps =
     types-requests
     psutil
     openai
+    anthropic[bedrock,vertex]
```

Comment:
Bedrock and Vertex clients are unsupported.

Reply:
I couldn't get it to run (locally) with just `anthropic` as dependency, IDK why. I declared bedrock & vertex coz I snatched that dependency configuration from anthropic vision demo own dependencies, which came as `anthropic[bedrock,vertex]`, and it worked....

Does it work for you without the extras?

Reply:
Anyway I think I added that dep because I was going to write tests which I never did in the end... We can remove the tox dep I think

Reply:
It won't work since the client and types are different for these. So better to remove this from the dependencies.